<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>611</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 63, 65);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  87 20pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>BIENVENIDO </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cocinero">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
background-color: #fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Imagenes proyecto/-chef.webp</normaloff>../../../Imagenes proyecto/-chef.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>81</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mesero">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../meserodesign-185162735.png</normaloff>../../../meserodesign-185162735.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>111</width>
      <height>111</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_adminLocal">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../AL.png</normaloff>../../../AL.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>85</width>
      <height>100</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_adminGeneral">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../imagen.png</normaloff>../../../imagen.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>92</width>
      <height>82</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../logoalfafood.png</normaloff>../../../logoalfafood.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>111</width>
      <height>111</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>SELECCIONE SU ROL PARA INGRESAR </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>COCINERO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>ADMINISTRADOR </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>295</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>GENERAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>MESERO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>ADMINISTRADOR </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>295</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 63, 65);
font:  8 8pt &quot;Arial Black&quot;; color: &quot;white&quot; ;
</string>
    </property>
    <property name="text">
     <string>LOCAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pedido">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
font: 87 8pt &quot;Arial Black&quot;; Color: &quot;white&quot;;
background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>   CREAR PEDIDO</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Imagenes proyecto/simbolomas.svg</normaloff>../../../Imagenes proyecto/simbolomas.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>71</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cerrar">
    <property name="geometry">
     <rect>
      <x>574</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F0F0F0;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
